libs/blocks/gnav/gnav-contextual-search.js

```diff
@@ -0,0 +1,87 @@
+import { fetchBlogArticleIndex } from '../article-feed/article-feed.js';
+import { getArticleTaxonomy, buildArticleCard } from '../article-feed/article-helpers.js';
+import { createTag } from '../../utils/utils.js';
+
+function highlightTextElements(terms, elements) {
+  elements.forEach((e) => {
```

Comment:
Nit: Prefer descriptive names for better readability
https://github.com/airbnb/javascript#naming-conventions